Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - default: false
+      description: The name of the address object.
+      isArray: false
+      name: name
+      required: true
+    - default: false
+      description: The address object's description.
+      isArray: false
+      name: description
+      required: false
+    - auto: PREDEFINED
+      default: false
+      description: The type of the address.
+      isArray: false
+      name: type
+      predefined:
+      - ip_netmask
+      - ip_range
+      - ip_wildcard
+      - fqdn
+      required: true
+    - default: false
+      description: A comma-separated list of address object tags.
+      isArray: true
+      name: tag
+      required: false
+    - default: false
+      description: The address value (should match the type)
+      isArray: false
+      name: address_value
+      required: true
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Create a new address object.
+    execution: false
+    name: prisma-sase-address-object-create
+    outputs:
+    - contextPath: PrismaSase.Address
+      description: Created address object.
+      type: String
+    - contextPath: PrismaSase.Address.description
+      description: Address description.
+      type: String
+    - contextPath: PrismaSase.Address.folder
+      description: Address folder.
+      type: String
+    - contextPath: PrismaSase.Address.id
+      description: Address ID.
+      type: String
+    - contextPath: PrismaSase.Address.type
+      description: Address type.
+      type: String
+    - contextPath: PrismaSase.Address.address_value
+      description: Address value.
+      type: String
+    - contextPath: PrismaSase.Address.name
+      description: Address name.
+      type: String
+  - arguments:
+    - default: false
+      description: ID of the address object to delete.
+      isArray: false
+      name: object_id
+      required: true
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Delete a specific address object.
+    execution: false
+    name: prisma-sase-address-object-delete
+  - arguments:
+    - default: false
+      description: The specific address object to return. If not specified, all addresses will be returned.
+      isArray: false
+      name: object_id
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or the type of folder?